### PR TITLE
Updating send json userids to endpoint in idLibrary

### PR DESCRIPTION
**Pull request by SKOCHERI**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Review by jdwieland8282 — APPROVED** [review]
LGTM

**Comment by robertrmartinez** [comment]
Needs second review I think, @blingster7 could you please?

File: modules/idLibrary.js
Comment (blingster7) [inline modules/idLibrary.js:203]: can we use `userIds` here as well?

Comment (SKOCHERI) [reply modules/idLibrary.js]: Updated from JSON.stringify(userIds) to userIds

**Review by blingster7 — APPROVED** [review]

**Pull request merged**
